src/kps/xml/objects/enums/Priority.java

```diff
@@ -31,31 +33,11 @@ public String toString() {
         }
     }
 
-    public boolean willSettleFor(Priority priority) {
-        if(this == DOMESTIC_STANDARD || this == INTERNATIONAL_STANDARD) {
-            return true;
-        }
-        if(this == DOMESTIC_AIR && priority != DOMESTIC_AIR) {
-            return false;
-        }
-        if(this == INTERNATIONAL_AIR && priority != INTERNATIONAL_AIR) {
-            return false;
-        }
-        return true;
-    }
-
-    public boolean willSettleFor(TransportType transportType) {
-        if(this == DOMESTIC_STANDARD || this == INTERNATIONAL_STANDARD) {
-            return true;
-        }
-        if((this == DOMESTIC_AIR || this == INTERNATIONAL_AIR) && transportType != TransportType.Air) {
-            return false;
-        }
-        return true;
-    }
-
     public boolean isDomestic() {
         return domestic;
     }
 
+    public boolean isAir() {
+        return air;
```

Comment:
It's mainly so that mail follows a route based on what the customers pay. It will chose a route with air delivery if that's what the customer pays, else it will just follow standard delivery (land and sea).